examples/getAllAccounts.js

```diff
@@ -18,7 +18,7 @@ dotenv.config();
       // indexer key
       token,
       // indexer url
-      'https://algoindexer.testnet.algoexplorerapi.io',
+      'https://mainnet-api.algonode.network',
```

Comment:
Is algoexplorer failing to do the boxes requests on these examples too?

Reply:
I hven't tried algoexplorer yet